reference/components/cf-monitord.markdown

```diff
@@ -120,6 +120,30 @@ into agent variables in the `$(mon.`name`)` context.
 
 Note: There is no way for force a refresh of the monitored data.
 
+## Statistical Classes
+
+`cf-monitord` automatically defines classes based on the observation of the data
+is has collected. Classes defined are named for the measurement id (the promise
+handle in the case of custom measurement promises) with prefixes and or suffixes
+depending on the measurement.
+
+The following suffixes may be used when defining classes:
+
+* `_high` :: The last measurement was greater than the average.
+* `_low` :: The last measurement was lower than the average.
```

Comment:
```suggestion
* `_low` :: The last measurement was low. It was lower than the average of all time and also lower than the recent average. This could indicate that the measured value is experiencing a "dip" or trending in a negative direction.
```